### PR TITLE
infoschema: Remove the unimplemented ALL_SQL_DIGESTS column from the DEADLOCKS table

### PR DESCRIPTION
**Pull request by MyonKeminta**
### What problem does this PR solve?

Problem Summary: The `ALL_SQL_DIGESTS` column of the `information_schema.deadlocks` table is not implement yet and we plan to release the first version without that column. The column has been added to the table but it content is always null. To avoid confusing the user, remove it for now and add it in the future when we implement the whole thing.

Ref: [Lock View](https://github.com/pingcap/tidb/issues/24199)

### What is changed and how it works?

What's Changed: The ALL_SQL_DIGESTS column  in `information_schema.deadlocks` is removed.

### Check List 

Tests 

- Unit test

Side effects

### Release note 

- No release note.

### CODE REVIEW

**Review by cfzjywxk — COMMENTED** [review]
/lgtm

File: util/deadlockhistory/deadlock_history.go [hunk 1]
```diff
@@ -148,12 +148,13 @@ func (d *DeadlockHistory) GetAllDatum() [][]types.Datum {
...
-				row[6] = "[" + strings.Join(item.AllSQLDigests, ", ") + "]"
-			}
+			row[6] = item.TxnHoldingLock
 
-			row[7] = item.TxnHoldingLock
+			// TODO: Implement the ALL_SQL_DIGESTS column for the deadlock table.
+			//row[7] = nil
+			//if item.AllSQLDigests != nil {
+			//	row[7] = "[" + strings.Join(item.AllSQLDigests, ", ") + "]"
+			//}
```
Comment (youjiali1995) [inline util/deadlockhistory/deadlock_history.go:157]: I prefer to delete them and comment out the column definition. 😕

Comment (MyonKeminta) [reply util/deadlockhistory/deadlock_history.go]: Updated. Is this what you mean?

**Review by youjiali1995 — APPROVED** [review]

**Pull request merged**